gptme/server/api_v2.py

```diff
@@ -23,6 +23,9 @@
 
 import flask
 from flask import request
+import tomlkit
```

Comment:
I think this file shouldn't need a `tomlkit` dependency, the toml stuff should be handled by config.py